apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -101,4 +165,9 @@ if (!('BridgeServiceMixin' in self)) {
       { methods: METHODS, streams: STREAMS, events: EVENTS }
     )
   );
+
+  // Automatically init service if it's run inside worker.
+  if ('importScripts' in self) {
```

Comment:
But how will we unit test this file then? I mean it's much more handy to call init explicitly in the tests....